docs/referenceConf.js

```diff
@@ -336,5 +336,10 @@ exports.config = {
 
   // Turns off source map support.  Stops protractor from registering global
   // variable `source-map-support`.  Defaults to `false`
-  skipSourceMapSupport: false
+  skipSourceMapSupport: false,
+
+  // Turns off WebDriver's environment variables overrides to ignore any
+  // environment variable and to only use the configuration in this file.
+  // Defaults to `true`
+  environmentOverrides: true
```

Comment:
Adding it here doesn't make it default to true, you need to change the default in https://github.com/angular/protractor/blob/master/lib/configParser.js.

But instead - let's name the variable `disableEnvironmentOverrides` and make it default to false. Having bools default to false is so much easier :)